.packaging/rust/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "enzyme"
+version = "0.4.0"
+edition = "2021"
+license = "MIT OR Apache-2.0"
+authors = [
+	"Manuel Drehwald",
+	"Lorenz Schmidt"
+]
+repository = "https://github.com/rust-ml/enzyme_build"
+keywords = ["enzyme", "auto-diff", "llvm", "machine-learning", "scientific-computing"]
+categories = ["development-tools::cargo-plugins", "mathematics", "science", "simulation"]
+readme = "README.md"
+documentation = "https://docs.rs/enzyme/"
+homepage = "https://github.com/rust-ml/enzyme_build"
```

Comment:
Left the docs site, since this one is auto-generated by the rust infra and we want the crate specific instructions.
Changed the homepage for enzyme.mit.edu, since the /rust isn't active yet.
I can adjust that later once we finished all the transitions.